src/live.asp.net/wwwroot/css/site.css

```diff
@@ -137,6 +137,9 @@ footer {
         font-size: 0.8em;
     }
 
+.label-new-show{
+    font-size: 14px;
```

Comment:
To be consistent with the rest of the file, please use a relative size rather than an exact pixel size here, e.g. `font-size: 0.8em`. You'll have to experiment to find which relative size works here.